tests/test_templates_layers.py

```diff
@@ -61,7 +61,7 @@ def weights(self):
                          [ 0.26999146, 0.26256351, 0.14722687, 0.23137066]])
             ]
 
-    def test_cvneuralnet_uses_correct_weights(self, gaussian_device_4modes, weights):
```

Comment:
Was this device never used?

Reply:
I think Johannes updated the tests with the OperationRecorder recently, so that we do not have to build qnodes!